### PR TITLE
Fl10 admin page

### PR DESCRIPTION
**Pull request by Evittta**

### CODE REVIEW

File: src/app/components/admin-page/admin-page.component.css [hunk 1]
```diff
@@ -0,0 +1,31 @@
...
+}
+.button-wrapper {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+}
+.button {
+  min-width: 100px;
+  margin: auto;
+  margin-top: 20px;
```
Comment (Danatlol) [inline src/app/components/admin-page/admin-page.component.css]: mayde margin: 20px auto auto;

File: src/app/components/admin-page/admin-page.component.css [hunk 2]
```diff
@@ -0,0 +1,31 @@
...
+}
+.mat-table {
+  overflow: auto;
+  max-height: 500px;
+}
+.mat-column-select {
+  overflow: visible;
+}
+.button-wrapper {
+  display: flex;
```
Comment (Danatlol) [inline src/app/components/admin-page/admin-page.component.css]: for flex try to use angular/flex-layout

File: src/app/components/admin-page/admin-page.component.css (hunk 1 above)
Comment (Danatlol) [inline src/app/components/admin-page/admin-page.component.css]: mayde margin: 20px auto auto;

File: src/app/components/admin-page/admin-page.component.html [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+    <input matInput (keyup)="applyFilter($event.target.value)" placeholder="Filter">
+  </mat-form-field>
+</div>
+<div class="example-container mat-elevation-z8" *ngIf="dataSource">
+  <mat-table #table [dataSource]="dataSource" matSort>
+    <ng-container matColumnDef="select">
+      <mat-header-cell *matHeaderCellDef>
+        <!--<mat-checkbox (change)="$event ? masterToggle() : null" [checked]="selection.hasValue() && 
+          isAllSelected()" [indeterminate]="selection.hasValue() && !isAllSelected()">
+        </mat-checkbox>-->
```
Comment (Danatlol) [inline src/app/components/admin-page/admin-page.component.html]: unnecessary comment

File: src/app/components/admin-page/admin-page.component.ts [hunk 4]
```diff
@@ -0,0 +1,82 @@
...
+      this.dataSource.sort = this.sort;
+    }, 100);
+  }
+  changeOrderStatus(order) {
+    if (order.status === 'new') {
+      order.status = 'accepted';
+      this.store.dispatch(new orderActions.UpdateOrder(order));
+    }
+  }
+  //ngAfterViewInit() {}
```
Comment (Danatlol) [inline src/app/components/admin-page/admin-page.component.ts]: comment

**Comment by Evittta** [comment]
[FL10 admin's page](https://trello.com/c/XJUru5jr/33-fl10-admins-page)

File: src/app/services/order.service.ts [hunk 5]
```diff
@@ -1,14 +1,35 @@
 import { Injectable } from '@angular/core';
-import { AngularFirestore } from 'angularfire2/firestore';
+import {
+  AngularFirestore,
+  AngularFirestoreCollection
+} from 'angularfire2/firestore';
 import { of } from 'rxjs/observable/of';
+import { Observable } from 'rxjs/Observable';
```
Comment (Danatlol) [inline src/app/services/order.service.ts]: use import { Observable } from 'rxjs' it's new import

**Pull request merged**